### PR TITLE
manifest v2

### PR DESCRIPTION
**Pull request by gabriel-okteto**

### CODE REVIEW

File: manifests/guestbook.yaml [hunk 1]
```diff
@@ -1,34 +1,54 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: guestbook
+  name: go-guestbook
```
Comment (pchico83) [inline manifests/guestbook.yaml]: Remove the `go-` prefix everywhere

File: okteto.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+build:
+  go-guestbook:
+    image: okteto.dev/go-guestbook:1.0.0
+    context: .
+
+deploy:
+  - kubectl apply -k manifests
```
Comment (pchico83) [inline okteto.yml:11]: let's keep `kustomize edit set image okteto/guestbook:golang=$OKTETO_BUILD_GUESTBOOK_IMAGE` and remove the image field from https://github.com/okteto/go-guestbook/pull/5/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70R3

File: manifests/guestbook.yaml (hunk 1 above)
Comment (gabriel-okteto) [reply manifests/guestbook.yaml]: I added those to fix? an issue where deployment is not found during okteto up

File: manifests/guestbook.yaml [hunk 3]
```diff
@@ -13,11 +12,11 @@ spec:
         app: guestbook
     spec:
       containers:
-      - image: okteto/guestbook:golang
-        name: guestbook
-        env:
-          - name: REDIS_HOST
-            value: redis
+        - image: okteto.dev/guestbook:1.0.0
```
Comment (pchico83) [inline manifests/guestbook.yaml:15]: Don't set `image`, or keep it empty. `kustomize` will set the value

File: manifests/kustomization.yaml [hunk 4]
```diff
@@ -2,5 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - guestbook.yaml
-- guestbook-public.yaml
-- redis.yaml
\ No newline at end of file
+- redis.yaml
```
Comment (pchico83) [inline manifests/kustomization.yaml:5]: do you need to commit this file? at least `gabriel-okteto` related values shouldn't be commited

File: okteto.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+build:
+  guestbook:
+    image: okteto.dev/guestbook:1.0.0
```
Comment (pchico83) [inline okteto.yml:3]: dont set the `image`, we will use `OKTETO_BUILD_GUESTBOOK_IMAGE` to let customize use the right image tag

**Review by pchico83 — APPROVED** [review]

File: okteto.yml (hunk 5 above)
Comment (gabriel-okteto) [reply okteto.yml:3]: re-added to fix the error

File: manifests/guestbook.yaml (hunk 3 above)
Comment (gabriel-okteto) [reply manifests/guestbook.yaml:15]: re-added after error

**Pull request merged**